designs/systems/index.md

```diff
@@ -0,0 +1,128 @@
+to. 
+
+There is a Condition dropdown which defines the current status for this system (Automated, Manual or Broken). If the status
+for a system is changed, the user needs to write down why it is changing. The text and timestamp will be displayed as the
+Last Condition Report. Beaker itself can change this status and update this text.
+
+The last card is Remove from Beaker. It has a button to completely remove the system from Beaker.
+
+![](img/system_details_settings.png)
+
+## System details - Power Options
+
+When a user clicks "Power Options" in the horizontal menu, all the information and configuration related to power is
+shown in the screen. 
+
+The "Power Control" card will tell if the system power can be controlled remotely and will have buttons to power the
+system on and off, reboot it, interrupt and clear netboot.
+
+A table with the 5 most recent power commands is also shown. There is a button for the user to see all of the power
+history for the machine. The table has 5 fields:
+- Date / Time: timestamp for the last status update for this power command
+- Trigger: identifies how the power command was issued
+- Status: Current status of the command
+- Action: the command itself
+- Message (optional): message returned by the command call, if any
+
+The "Power Settings" card will let the user set the power configuration for the system, as address, protocol, ID, username,
+password, etc.
+
+![](img/system_details_power.png)
+
+## System details - Provisioning
+
+When the "Provisioning" option is selected in the horizontal menu, there will be two cards in the screen.
+
+One of the cards will have the options to provision the system with a specific distro from the library. The user will be
+able to select the family, tag, distro and tree as well as a set of Install Options to provision the system. A checkbox
+may be selected to reboot the system.
+
+Install Options can be defined in the second card. The "default" set of options will always be available, for all families
+and architectures, with specific KS metadata and Kernel options for the installation. The user may click the "Add" button
+to create customized sets of options for specific families and / or architectures supported by the system, and edit them
+afterwards. Clicking the trash can icon for a set of options excludes it.
+
+![](img/system_details_provision.png)
+
+## System details - Permissions
+
+When the "Permissions" option is selected in the horizontal menu, a card is shown with the specific permissions for the
+machine. There are two main options: apply the policy according to a previously defined pool or apply a custom policy.
+
+If the custom policy is selected, a default group called "Everyone" is created. More users and/or groups may be added
+by clicking an "Add" button.
+
+For every user / group defined in the policy, all specific permissions will be shown with checkboxes. Marking a checkbox
```

Comment:
For each user?  What happens when you run out of space on the screen.  I'm wondering if original format is better.

Reply:
For each user defined in the policy. Most systems would have the policy defined by group, or a specific set of users.
If a system particularly has a complex set of permissions, then the number would grow and a srcoll bar would be there as well... Not sure on how yet.
The original format is just ugly I would say, but we can revert back to it if there is not a viable solution after all. As Daniel said, this screen layout really has room for improvement.